src/app/services/notifications.service.ts

```diff
@@ -15,6 +15,12 @@ export class NotificationsService {
   private environment = environment;
   private _notifications: ReplaySubject<Array<INotification>> = new ReplaySubject<Array<INotification>>(1);
   public readonly  notifications: Observable<Array<INotification>> = this._notifications.asObservable();
+  private _loadingFeatureData: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Comment:
> I wasn't sure where the best place to put the loading state. So, I just put it in notifications.service.

what about in the `geo-data.service.ts`?   then this loading variables will with the related get commands